src/main/resources/hudson/plugins/gradle/injection/InjectionConfig/config.jelly

```diff
@@ -98,6 +98,9 @@
                         </f:entry>
                     </f:repeatableProperty>
                 </f:entry>
+                <f:entry field="gradleCaptureTaskInputFiles" help="/plugin/gradle/help-captureFileInputs.html">
```

Comment:
I think you can avoid adding the `help` attribute and just name by convention the tooltip file `help-gradleCaptureTaskInputFiles.html`

Reply:
Yup, works, had to move them to a different folder though